config/locales/idv/fr.yml

```diff
@@ -35,6 +35,7 @@ fr:
         de cet état.
       unsupported_jurisdiction_sp: Veuillez visiter %{sp_name} pour accéder à votre
         compte.
+      unsupported_otp_delivery_method: Sélectionnez une méthode pour recevoir un code
```

Comment:
Minor issue: this is missing a period at the end compared to the other languages.